### DISCUSSION TITLE
Question on solving issue #2419 pull from __all__ exports

### DISCUSSION THREAD

**Discussion by aciffone23** [perspective-dev/perspective#2476] [General]
I'm a first time contributor. I wanted to pick this [issue](https://github.com/finos/perspective/issues/2419) up. Im wondering if there are more details on this. Can you point to a few files where `_all_` isn't defined?

**Comment by timkpaine** [comment]
Looks like it's mostly just the `handlers` folder that needs to be dealt with.
